src/components/configure/header/Header.tsx

```diff
@@ -105,112 +116,126 @@ export default class Header extends React.Component<HeaderProps, HeaderState> {
+                        disabled={isOpenedKbd}
+                      >
+                        <div className="device-item">
+                          {isOpenedKbd && (
+                            <Link
+                              fontSize="small"
+                              className="link-icon link-on"
+                            />
+                          )}
+                          <div className={['device-name', linking].join(' ')}>
+                            {info.productName}
+                            <span className="device-ids">
+                              (VID: {hexadecimal(info.vendorId, 4)} / PID:{' '}
+                              {hexadecimal(info.productId, 4)})
+                            </span>
+                          </div>
+                        </div>
+                      </MenuItem>
+                    );
+                  })}
+
                   <MenuItem
-                    key={index}
-                    onClick={
-                      isOpenedKbd
-                        ? undefined
-                        : this.onClickKeyboardMenuItem.bind(this, kbd)
-                    }
-                    disabled={isOpenedKbd}
+                    key="another"
+                    onClick={this.onClickConnectionMenuItemNewDevice.bind(this)}
                   >
-                    <div className="device-item">
-                      {isOpenedKbd && (
-                        <Link fontSize="small" className="link-icon link-on" />
-                      )}
-                      <div className={['device-name', linking].join(' ')}>
-                        {info.productName}
-                        <span className="device-ids">
-                          (VID: {hexadecimal(info.vendorId, 4)} / PID:{' '}
-                          {hexadecimal(info.productId, 4)})
-                        </span>
-                      </div>
+                    <div className="another-device-wrapper">
+                      <Button
+                        size="small"
+                        color="primary"
+                        className="another-device"
+                      >
+                        + KEYBOARD
+                      </Button>
                     </div>
                   </MenuItem>
-                );
-              })}
-
-              <MenuItem
-                key="another"
-                onClick={this.onClickConnectionMenuItemNewDevice.bind(this)}
-              >
-                <div className="another-device-wrapper">
-                  <Button
-                    size="small"
-                    color="primary"
-                    className="another-device"
-                  >
-                    + KEYBOARD
-                  </Button>
-                </div>
-              </MenuItem>
-            </Menu>
+                </Menu>
+              </div>
+            </div>
+            <IconButton onClick={this.onClickShowInfoDialog.bind(this)}>
```

Comment:
It's huge diff because I wrapped the div tag for adjusting layout.
Except it, the actual diff is only this and the <InfoDialog> on the last part of this diff.